GraphQL.Annotations.TSql/SqlFieldMutator.cs

```diff
@@ -41,7 +41,7 @@ public static class SqlFieldMutator
 				.Where(v => v.Attribute.ReturnType != typeof(IdGraphType))
 				.ToList();
 			var notIdentity = fields
-				.Where(v => !v.Attribute.IsIdentityField)
+				.Where(v => !v.Attribute.SkipOnInsert)
```

Comment:
consolidate where clause

Reply:
This where clause cannot be consolidated since it is the only one